components/SensorComponent.js

```diff
@@ -2,95 +2,107 @@ import React, { useState, useEffect } from "react";
 import { Accelerometer } from "expo-sensors";
 import styled from "styled-components/native";
 
-// ==========================
-// = Functions
-const isShaking = (data) => {
-  // x,y,z CAN be negative, force is directional
-  // We take the absolute value and add them together
-  // This gives us the total combined force on the device
-  const totalForce = Math.abs(data.x) + Math.abs(data.y) + Math.abs(data.z);
+import { URL } from "./utils/urls";
 
-  // If this force exceeds some threshold, return true, otherwise false
-  // Increase this threshold if you need your user to shake harder
-  return totalForce > 1.78;
-};
-
-// ==========================
-// = Styled components
-const ShakeView = styled.View`
-  display: flex;
-  flex-direction: column;
+const Container = styled.View`
+  flex: 3;
+  justify-content: center;
+  align-items: center;
+  margin: 40px;
+  padding: 0;
 `;
-
-const ShakeAlert = styled.Text`
-  font-size: 36px;
-  font-weight: bold;
-  color: #aa0000;
+const Quote = styled.Text`
+  font-size: 35;
+  line-height: 45;
+  text-align: left;
 `;
-const ShakeDataView = styled.View``;
-const ShakeDataTitle = styled.Text`
-  font-weight: bold;
+const AuthorContainer = styled.View`
+  flex-direction: row;
+  text-align: left;
+  width: 300px;
+  margin: 15px;
 `;
-const ShakeData = styled.Text``;
+const Author = styled.Text`
+  font-style: italic;
+  font-size: 15;
+`;
+const Button = styled.TouchableOpacity`
```

Comment:
Nice use of TouchableOpacity, I discovered it after I already added a bunch of buttons in my project :(